client/pages/offer-thread.jsx

```diff
@@ -1,42 +1,66 @@
 
 import React from 'react';
 import io from 'socket.io-client';
+let buyerId = null;
+let sellerId = null;
```

Comment:
You should be able to delete these variables, or at least update your code so they're not necessary.